cicecore/shared/ice_init_column.F90

```diff
@@ -280,6 +281,10 @@ subroutine init_shortwave
             albpnd(i,j,iblk) = c0
             snowfrac(i,j,iblk) = c0
             apeff_ai(i,j,iblk) = c0
+            swuvrdr(i,j,iblk) = c0
+            swuvrdf(i,j,iblk) = c0
+            swpardr(i,j,iblk) = c0
+            swpardf(i,j,iblk) = c0
```

Comment:
Why is this needed here, but we don't initialize the visible and infrared terms to zero here?  Is it because the visible and infrared shortwave are always set to something non-zero elsewhere but the uvr and par terms are not (if they aren't used?)  Should we set swvdr, swvdf, swidr, and swidf = c0 here too?

Reply:
These 4 terms are for GEOS only and I noticed they  are not being initialized to zero after being allocated. So I figured here is the place to do it. The terms (vis and nir) already in CICE presumably exist in restart or initialized somewhere.